src/Backend/Core/Layout/Sass/components/_badges.scss

```diff
@@ -7,20 +7,27 @@
 
 .strength-none.badge {
```

Comment:
I find it a bit odd that there are seperate badge-names for this. Wouldn't it be easier to use `.badge-danger`, `.badge-warning`,...?

Reply:
I am agree with this, but I think it wasn't that easy cause of the way it's implemented, but I will check this to be sure.